tests/storage/fsp/conftest.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 # Explicitly import package-scoped fixtures (see explanation in pkgfixtures.py)
-from pkgfixtures import host_with_saved_yum_state
+from pkgfixtures import __host_with_saved_yum_state_cm
```

Comment:
`pkgfixtures` would not be the place for a cm.  If it is meant to be shared, then it should go into `lib/`, and not start with underscores.